game/forms.py

```
@@ -29,3 +30,26 @@ def clean(self):
+        to_email, 
+        html_email_template_name=None
+    ):
+        try:
+            super().send_mail(
+                subject_template_name, 
+                email_template_name, 
+                context, 
+                from_email, 
+                to_email,
+                html_email_template_name)
+        except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)